### ISSUE TITLE
Clarify --config option (provide example)

### ISSUE THREAD

**Issue by andykog** [nwolverson/purescript-language-server#57]
Hey, I'm trying to use this language server with vim. According to README, the flag --config is required, but there is no explanation on what should the value contain. There is a link to vscode-ide-purescript's package.json but it doesn't help much. Would be great if you provided some example config.

**Comment by andykog** [comment]
Managed to get it working, turned out this flag is optional. So the following paragraph is a bit misleading (I'd replace `should` -> `may`):

> Currently you should pass a --config flag followed by a JSON object, or immediately send config on starting the server in the same way VS Code does - this should improve in future

**Comment by nwolverson** [comment]
Well, that does say pass config or send config on starting, so I presume either the vim stuff is now sending config or I changed something, we should figure that out.

Basically there is some config that is only read once at startup, and there is a mechanism in the LSP to send config, vscode always sent this at the start so we waited for that to be sent before proceeding, unless it was passed on the command line.

Would be good to reevaluate this.

Regarding the actual content, the possible values are basically here https://github.com/nwolverson/vscode-ide-purescript/blob/master/package.json#L77 but it would definitely be useful to have an example of what this looks like in JSON. The only config I could confidently give from memory is this one: `{}` :smile:
